scripts/source/_04_make_forward.py

```diff
@@ -119,6 +119,15 @@ def run_forward(*, cfg, subject, session, in_files):
 
     # Info
     info = mne.io.read_info(in_files.pop('info'))
+    info = mne.pick_info(
+        info,
+        mne.pick_types(
+            info,
+            meg="meg" in cfg.ch_types,
```

Comment:
IIRC it has to be more than this I think since `ch_types` can also be `'mag'` or `'grad'` or lists of these